extensions/transport/portability-transport-jettyrest/src/main/java/org/datatransferproject/transport/jettyrest/http/JettyMonitor.java

```diff
@@ -60,7 +60,7 @@ public void debug(Throwable thrown) {
   }
 
   public void debug(String msg, Throwable thrown) {
-    DELEGATE.info(() -> msg, thrown);
+    DELEGATE.debug(() -> msg, thrown);
```

Comment:
Maybe there is a reason for this being escalated to info from debug. If so I'm happy to add a comment explaining why.

Reply:
I think it was a cut-and-paste error :-)